tasks/loopback_angular.js

```diff
@@ -45,6 +46,9 @@ module.exports = function(grunt) {
     if (!options.output)
       grunt.fail.warn('Missing mandatory option "output".');
 
+    if(!options.modelsToIgnore)
+      grunt.log.writeln('No models to ingnore, Generating endpoints for each models');
```

Comment:
I prefer the tools to be quite and print to console only when there is something needing my attention.

Please revert this change too.